bin/bashrc

```diff
@@ -40,9 +48,6 @@ module load desiconda/20180512-1.2.5-img
 # local astrometry.net install?
 # PYTHONPATH=${CODE_DIR}/astrometry/lib/python:${PYTHONPATH}
 
-export PYTHONPATH
-export PATH
-
 export QDO_BATCH_PROFILE=$NERSC_HOST
 export QDO_BACKEND=postgres
 export QDO_DB_NAME=desirun
```

Comment:
Didn't we agree the QDO stuff should go into desiproc's .bashrc.ext file and not here?  Fine either way.